src/fusets/mogrp.py

```diff
@@ -0,0 +1,138 @@
+import itertools
+
+import GPy
+import numpy as np
+
+
+def create_empty_2D_list(nrows, ncols):
```

Comment:
@msalinero : is it necessary to define this function? can't we just use `np.zeros(nrows, ncols)` or a python oneliner like `[[0] * ncols for _ in range(nrows)]` ?

Reply:
@soxofaan So sorry for the very very late response. Indeed it is not necessary. `[[0] * ncols for _ in range(nrows)]` is tested and works fine.